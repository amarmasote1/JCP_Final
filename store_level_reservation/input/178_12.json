[     {
	
	"storeId":"2098",    
      "shutdownIndi":"P",     
      "storeShutdown":"Y",     
      "startDate":"<$yesterday>",    
      "endDate":"<$nextWeek>"
	
},
{
	
	"storeId":"2098",    
      "shutdownIndi":"S",     
      "storeShutdown":"Y",     
      "startDate":"<$yesterday>",    
      "endDate":"<$nextWeek>"
	
}


]