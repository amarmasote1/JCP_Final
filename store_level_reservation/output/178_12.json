{
    "body": [
        {
            "warnings": null,
            "message": "Data Write Successful",
            "body": null,
            "key": "2098|P",
            "status": 212
        },
        {
            "warnings": null,
            "message": "Data Write Successful",
            "body": null,
            "key": "2098|S",
            "status": 212
        }
    ],
    "message": "Successful",
    "status": 221
}