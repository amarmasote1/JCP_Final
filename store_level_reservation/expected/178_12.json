{"body":[{"status":212,"message":"Data Write Successful","key":"2098|P","body":null,"warnings":null},{"status":212,"message":"Data Write Successful","key":"2098|S","body":null,"warnings":null}],"status":221,"message":"Successful"}